miasm2/jitter/JitCore.c

```diff
@@ -156,26 +155,6 @@ void MEM_LOOKUP_INT_BN_TO_PTR(JitCpu* jitcpu, int size, uint64_t addr, char* ptr
 	memcpy(ptr, (char*)&ret, size / 8);
 }
 
-void __attribute__((weak)) MEM_WRITE_08(JitCpu* jitcpu, uint64_t addr, uint8_t src)
```

Comment:
Hum. The use of the weak attribute was to allow default symbol (with default dehavior) for the mem_lookup_xx and mem_write_xx.
In some 'special' architectures, the symbol may be redefined: For example, in arm, we do not want to flush the jitted blocks (ie instruction cache) directly after a memory write.

The problem here is my fault: when I modified the API to use bignum, I totally forgotten to use the weak attribute for the new api, which doesn't use those functions anymore (mem_write_XX) but uses mem_write_bn_xxx instead.

So here, on x86 the behavior is ok, but it may fail on arm architecture.
I whish I had written a regression test for the ARM automodifying code...

I will look at this closely at the end of the week!

Reply:
Let me know what you want me to do here :). There's not a real equivalent AFAIK in MSVC but there's https://msdn.microsoft.com/en-us/library/5tkz6s71.aspx that could help, maybe :).

Reply:
I will have a look at this in the next days: I have to do the pro&cons between weak symbols and codegen support... Next weekend may bring us the answer :smiley: